doc/source/admin/galaxy_options.rst

```diff
@@ -477,6 +477,20 @@
 :Type: str
 
 
+~~~~~~~~~~~~~~~~~~~~~
+``watch_core_config``
+~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    Set to true to enable monitoring of a subset of options in the
+    core configuration file (See RELOADABLE_CONFIG_OPTIONS in
+    lib/galaxy/config/__init__.py). If changes are found, modified
+    options are automatically reloaded. Takes the same values as the
+    'watch_tools' option.
+:Default: ``false``
+:Type: str
```

Comment:
```suggestion
:Type: bool
```

Reply:
Oh, I see now. 'watch_tools' accepts more than true/false, and we use the same range of values.